caffe2/python/modeling/parameter_info.py

```diff
@@ -15,7 +15,7 @@ class ParameterType(object):
 class ParameterInfo(object):
     def __init__(
             self, param_id, param, key=None, shape=None, length=None,
-            grad=None):
+            grad=None, blob_copy=None):
```

Comment:
could you add a comment about what blob_copy is for? Also how about calling it "float_copy" ?